Fastfile.private

```diff
@@ -204,36 +229,44 @@ reflected in `Info.plist` during building."
   desc "Keep these new certificates and profiles in sync with Match repository"
   private_lane :create_app do |options|
 
+    build_configuration_key = options[:build_configuration]
+    build_configuration = get_build_configuration(build_configuration: build_configuration_key)
+    app_name = get_application_name(build_configuration: build_configuration_key) % project_name
+    skip_itc = build_configuration_key == :test
```

Comment:
Always thought the build_configuration_key was like a string. Gotta get more acquaintanced with ruby.

Reply:
Wait, why wouldn't it? `:test` maps to a string while `skip_itc` is a boolean

Reply:
But :test and "test" is not the same, right?

Reply:
when I use the suffix `key`, is because I'm referring to the symbol (`:test`), if I use it without `key`, I'm  referring to the string `test`.

They're not the same.